README.Rmd

```diff
@@ -32,6 +32,58 @@ Check out the [HTTP testing book](https://books.ropensci.org/http-testing) and t
 * [crul](https://github.com/ropensci/crul)
 * [httr](https://github.com/r-lib/httr)
 
+## Getting Started
+
+To install from CRAN
+
+```r
+install.packages("vcr")
+```
+
+You can then set up your package to use VCR with:
+
+```r
+vcr::use_vcr()
+```
+
+This will:
+
+```
+◉ Using package: vcr.example  
+◉ assuming fixtures at: tests/fixtures  
+✓ Adding vcr to Suggests field in DESCRIPTION  
+✓ Creating directory: ./tests/testthat  
+◉ Looking for testthat.R file or similar  
+✓ tests/testthat.R: added  
+✓ Adding vcr config to tests/testthat/helper-vcr.example.R  
+✓ Adding example test file tests/testthat/test-vcr_example.R  
+✓ .gitattributes: added  
+◉ Learn more about `vcr`: https://books.ropensci.org/http-testing
+```
+
+If you need to use api keys for authentication, it's important to protect them. `vcr` saves responses from API's as YAML files, and this will include your api keys unless you indicate to `vcr` what they are and how to protect them. This is one of the uses  of the `helper-vrc` file that got made for us just now. By default it will look like this:
```

Comment:
can you say instead:

> secrets (for example, api keys for authentication), ...

And then use the term secret or secrets because it's not just for API keys as you'd imagine, but any secrets the user does not want the internet to know about (api keys, username, email, password, etc.)